CONTRIBUTING.md

```diff
@@ -46,6 +46,34 @@ Your ideas, documentation and code have become an integral part of this project.
 
 In fact, feel free to open a PR to add your name to the [`AUTHORS`](AUTHORS.md) file and get eternal attribution.
 
+## Releasing a new version
+
+0. Review state of the 'develop' branch
+    - Ensure all changes intended for release are merged
+1. Create a release branch
+    - From 'develop', git checkout -b "release-$MAJOR.$MINOR.$PATCH"
+2. Update the new release
+    - [ ] Update version number in `_config.yml` and `README.md`
+    - [ ] Update [`AUTHORS.md`](AUTHORS.md) with new contributors
+    - [ ] Update [`CHANGELOG.md`](CHANGELOG.md)
+    - [ ] Perform extra pass on diff to the 'master' branch
+        - Reread any section or paragraph to ensure wording changes still fit the whole and do not contain grammar or spelling errors
+        - If needed, commit changes and repeat extra pass
+    - [ ] Push branch, open a pull request to the 'master' branch
+        - Request review from multiple reviewers, especialy a proofreader
+    - [ ] Generate new PDF of the primary content
+        - Serve html content with `script/serve.sh`
+        - In a browser, navigate to http://127.0.0.1:4000/
```

Comment:
```suggestion
        - You can validate the server is running by navigating to http://127.0.0.1:4000/ in your browser
```

Reply:
Ah, well, I just pushed a similar change. Let me know which wording you prefer.